From d509e79d3fc0dbb716e7b1a6cc504a9bc1addcf8 Mon Sep 17 00:00:00 2001
From: LateDreamXD <127290936+LateDreamXD@users.noreply.github.com>
Date: Fri, 20 Feb 2026 11:42:19 +0800
Subject: [PATCH] patch: ExtensionManager.vue

---
 src/components/ExtensionManager.vue | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/ExtensionManager.vue b/src/components/ExtensionManager.vue
index 21213f8..bbf045b 100644
--- a/src/components/ExtensionManager.vue
+++ b/src/components/ExtensionManager.vue
@@ -75,7 +75,7 @@
         <div class="empty-icon">
             <i class="fas fa-puzzle-piece"></i>
         </div>
-        <h4>{{ t('web-cha-jian-ti-shi') }}</h4>
+        <h4>当前版本暂不支持插件</h4>
     </div>
 </template>
 
-- 
2.52.0.windows.1

