From 2fe38f6acf68c9ffe420bf404187423b06f5cc1d Mon Sep 17 00:00:00 2001
From: LateDreamXD <127290936+LateDreamXD@users.noreply.github.com>
Date: Fri, 20 Feb 2026 13:43:04 +0800
Subject: [PATCH] patch: vite.config.js

---
 vite.config.js | 79 +-------------------------------------------------
 1 file changed, 1 insertion(+), 78 deletions(-)

diff --git a/vite.config.js b/vite.config.js
index 8cb59ef..8691b42 100644
--- a/vite.config.js
+++ b/vite.config.js
@@ -1,87 +1,10 @@
 // vite.config.js
 import { defineConfig } from 'vite';
 import vue from '@vitejs/plugin-vue';
-import { VitePWA } from 'vite-plugin-pwa';
 
 export default defineConfig({
   plugins: [
-    vue(),
-    VitePWA({
-      registerType: 'autoUpdate',
-      devOptions: {
-        enabled: true,
-        suppressWarnings: true,
-        suppressErrors: true,
-        logLevel: 'silent',
-        type: 'module'
-      },
-      workbox: {
-        // 缓存策略配置
-        runtimeCaching: [
-          {
-            // 匹配所有API请求
-            urlPattern: /^\/api\/.*$/i,
-            handler: 'NetworkFirst',
-            options: {
-              cacheName: 'api-cache',
-              expiration: {
-                maxEntries: 100,
-                maxAgeSeconds: 60 * 60 * 24 * 1
-              },
-              cacheableResponse: {
-                statuses: [0, 200]
-              }
-            }
-          },
-          {
-            // 匹配所有静态资源
-            urlPattern: /\.(?:png|jpg|jpeg|svg|gif|webp|js|css|woff2)$/i,
-            handler: 'StaleWhileRevalidate',
-            options: {
-              cacheName: 'assets-cache',
-              expiration: {
-                maxEntries: 100,
-                maxAgeSeconds: 60 * 60 * 24 * 7
-              }
-            }
-          },
-          {
-            // 匹配导航请求（HTML）
-            urlPattern: /\/$/,
-            handler: 'NetworkFirst',
-            options: {
-              cacheName: 'html-cache',
-              expiration: {
-                maxEntries: 10,
-                maxAgeSeconds: 60 * 60 * 24 // 1天
-              }
-            }
-          }
-        ]
-      },
-      manifest: {
-        name: 'MoeKoe 萌音',
-        short_name: 'MoeKoe',
-        description: '一个高颜值的kugou第三方播放器',
-        theme_color: '#FF69B4',
-        background_color: '#ffffff',
-        display: 'standalone',
-        start_url: '/',
-        display_override: [
-          "window-controls-overlay",
-          "standalone",
-          "minimal-ui",
-          "browser"
-        ],
-        icons: [
-          {
-            src: '/assets/images/logo.png',
-            sizes: '256x256',
-            type: 'image/png'
-          }
-        ]
-      }
-    })
+    vue()
   ],
   base: '',
   server: {
-- 
2.52.0.windows.1

